run/events-gcs/anthos.md

```diff
@@ -0,0 +1,100 @@
+# Events for Cloud Run on Anthos – GCS tutorial
+
+This sample shows how to create a service that processes GCS events on Anthos.
+It is assumed that you already created a GKE cluster with Events for Cloud Run
```

Comment:
nit: Consider adding links here to instructions if possible.

Reply:
Yes, I did consider but the docs are not ready yet. We have an alpha user guide, but not sure how long that will stay around. Ideally, beta docs are published soon and we can link to them soon after.

Reply:
@amanda-tarafa Please let me know if you still want me to link to alpha user guide. My vote would be to wait for the beta docs but either way is fine with me.

Reply:
Nope, that's fine, I approved the PR because this was a non-bloker. I'll merge now, given that there are unrelated failures on CI.